src/middlewared/middlewared/alert/source/memory_errors.py

```diff
@@ -10,7 +11,16 @@ class MemoryErrorsAlertClass(AlertClass):
     proactive_support = True
 
 
+class MemorySizeMismatchAlertClass(AlertClass):
+    category = AlertCategory.HARDWARE
+    title = 'Memory Size Mismatch Detected'
+    text = 'Memory size on this controller %(r1)d doesn\'t match other controller %(r2)d'
```

Comment:
Should we format these numbers using `humanfriendly`?

Reply:
Great question, I went back and forth in my head about this and still don't know what I want to do 😄 I guess i'll use your review as an excuse to format the numbers.